test/runtests.jl

```diff
@@ -31,8 +31,13 @@ end
   include("compiler.jl")
 end
 
-@testset "CUDA tests" begin
-  include("cuda.jl")
+@info "Starting GPU intgeration tests..."
+if haskey(ENV, "CI_GITLAB_CUDA")
```

Comment:
You can use CUDAapi to check that CUDA is set up – then GPU tests will run locally if cuda is available, too.

Reply:
Got my answer 👍